plugins/modules/win_webpicmd.py

```diff
@@ -18,7 +18,7 @@
 description:
     - Installs packages using Web Platform Installer command-line
       (U(http://www.iis.net/learn/install/web-platform-installer/web-platform-installer-v4-command-line-webpicmdexe-rtw-release)).
-    - Must be installed and present in PATH (see M(win_chocolatey) module; 'webpicmd' is the package name, and you must install 'lessmsi' first too)?
+    - Must be installed and present in PATH (see M(chocolatey.chocolatey.win_chocolatey) module; 'webpicmd' is the package name, and you must install 'lessmsi' first too)?
```

Comment:
This line now is a bit too long, will need to make sure we don't exceed 160 chars to keep the sanity check happy.